packages/nodes-base/nodes/CustomerIo/GenericFunctions.ts

```diff
@@ -0,0 +1,48 @@
+import {
+	IExecuteFunctions,
+	IHookFunctions,
+	ILoadOptionsFunctions,
+} from 'n8n-core';
+
+import { OptionsWithUri } from 'request';
+import { IDataObject } from 'n8n-workflow';
+
+export async function apiRequest(this: IHookFunctions | IExecuteFunctions | ILoadOptionsFunctions, method: string, endpoint: string, body: object, query?: IDataObject): Promise<any> { // tslint:disable-line:no-any
+	const credentials = this.getCredentials('customerIoApi');
+
+	if (credentials === undefined) {
+		throw new Error('No credentials got returned!');
+	}
+
+	query = query || {};
+
+	const options: OptionsWithUri = {
+		headers: {
+			'Content-Type': 'application/json',
+			'Authorization': `Bearer ${credentials.apiKey}`,
+		},
+		method,
+		body,
+		qs: query,
```

Comment:
name the param qs so that you can write `qs,`

Reply:
yes!